core/src/rpc.rs

```diff
@@ -738,7 +740,12 @@ impl JsonRpcRequestProcessor {
                         bigtable_blocks.retain(|&slot| slot <= end_slot);
                         bigtable_blocks
                     })
-                    .unwrap_or_else(|_| vec![]));
+                    .map_err(|_| {
+                        Error::invalid_params(
```

Comment:
Oops, this decent error doesn't translate to bad http reqs to my surprise... I'll just leave this as-is... ;)

```
$ date && curl -v -H 'Content-Type: application/json' -d '{"jsonrpc":"2.0","id":1,"method":"getConfirmedBlocks","params":[50000000,50040001]}' http://localhost:8899 ; date
Tue Jan 19 06:37:03 UTC 2021
*   Trying 127.0.0.1:8899...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 8899 (#0)
> POST / HTTP/1.1
> Host: localhost:8899
> User-Agent: curl/7.68.0
> Accept: */*
> Content-Type: application/json
> Content-Length: 83
> 
* upload completely sent off: 83 out of 83 bytes
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< content-type: application/json; charset=utf-8
< content-length: 123
< date: Tue, 19 Jan 2021 06:37:03 GMT
< 
{"jsonrpc":"2.0","error":{"code":-32602,"message":"BigTable query failed (maybe timeout due to too large range?)"},"id":1}
* Connection #0 to host localhost left intact
Tue Jan 19 06:37:33 UTC 2021

```